src/SingleCellAnnotator.imjoy.html

```diff
@@ -1437,7 +1438,7 @@
           </div>
             <br>
           <div class="btn-group">
-            <button :disabled="current_image_data.location_changed || current_image_data.discarded || current_image_data.unsure" class="btn transparent-btn" @click="confirmAnnotation()">&nbsp;Confirm</button>
+            <button :disabled="current_image_data.discarded || current_image_data.unsure" class="btn transparent-btn" @click="confirmAnnotation()">&nbsp;Confirm</button>
```

Comment:
Should we change the Confirm to Save to be consistent with the save button in segmentation? This button will not only confirm unchanged labels, but also force save existing changed labels.

Reply:
sure

Reply:
changed the button name and func name accordingly